lang.go

```diff
@@ -627,6 +634,7 @@ var Langs = map[Lang]string{
 	Ceb: "Cebuano",
 	Ces: "Czech",
 	Cmn: "Mandarin",
+	Yue: "Cantonese",
```

Comment:
The languages are in order alphabetically (in English). This line should go above `Zul: "Zulu"`.